skipruntime-ts/core/src/skipruntime_api.ts

```diff
@@ -160,26 +158,24 @@ export interface NonEmptyIterator<T> extends Iterable<T> {
   uniqueValue: () => Opt<T>;
 
   /**
-   * Returns an array containing the remaining values of the iterator
+   * Returns an array containing all values of the iterator
    */
   toArray: () => T[];
 
   /**
    * Performs the specified action for each element in the iterator.
-   * @param callbackfn - A function that accepts up to thow arguments. forEach calls the callbackfn function one time for each element.
-   * @param thisArg - An object to which the this keyword can refer in the callbackfn function. If thisArg is omitted, undefined is used as the this value.
+   * @param f - A callback to invoke on each element
+   * @param thisObj - An object to bind as `this` within the `f` invocations; if omitted, `undefined` is used
```

Comment:
Is "if omitted, `undefined` is used" exactly the behavior of optional arguments in TS? If so, if we explicitly mention it here, will users be confused thinking that since we're mentioning it there must be something nonstandard happening, but they can't see what it is? That is at least what happens for me.